code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/AuthIntegrationManager.java

```diff
@@ -0,0 +1,48 @@
+package io.cattle.platform.iaas.api.auth;
```

Comment:
As discussed, this class is incomplete and unused currently by the UI.
Delete this class if @vincent99 can live with /ldapconfig and /githubconfig instead of a single /authconfig endpoint.